pkgs/top-level/all-packages.nix

```diff
@@ -10777,6 +10781,10 @@ in
     inherit (darwin) IOKit;
   };
 
+  libdvdcss12 = callPackage ../development/libraries/libdvdcss/1.2.nix {
```

Comment:
I'm not real keen on adding a second version of libdvdcss, and one which is no longer maintained at that. It would be much better to find a way to make cd imgools work with the current, maintained, version

Reply:
Got it, thanks for the review! Hope to see what patching `cdimgtools` to use latest `libdvdcss` involves.

Reply:
Thank G-d, managed to patch `cdimgtools` to work with latest `libdvdcss`.